metrics/metrics.py

```diff
@@ -28,13 +31,23 @@ def __init__(self, url, headers, offset_minutes, max_retries, logger) -> None:
         self.url = url
         self.max_retries = max_retries
         self.logger = logger
+        self.client_id = client_id
+        self.csrf_enabled = csrf_enabled
 
     def collect(self):
         """
         Get and set Prefect work queues metrics.
 
         """
         ##
+        # PREFECT GET CSRF TOKEN IF ENABLED
+        #
+        if self.csrf_enabled == "True":
```

Comment:
if you make it a boolean before you pass it into the class, then you can just do:

```suggestion
        if self.csrf_enabled:
```